go-controller/hybrid-overlay/cmd/hybrid-overlay-node/hybrid-overlay-node.go

```diff
@@ -38,22 +38,33 @@ func main() {
 			Destination: &nodeName,
 		},
 		&cli.BoolFlag{
-			Name:        "windows-service",
-			Usage:       "Enables hybrid overlay to run as a Windows service. Ignored on Linux.",
-			Destination: &runAsWindowsService,
+			Name:  windowServiceArgName,
+			Usage: "Enables hybrid overlay to run as a Windows service. Ignored on Linux.",
 		}})
+
+	ctx := context.Background()
+
 	c.Action = func(c *cli.Context) error {
+		if err := initForOS(c, &ctx); err != nil {
```

Comment:
@mansikulkarni96 don't need the & for context here either, eg initForOS() should just take a `context.Context` without a *

Reply:
done